pkg/florist/netstat.go

```diff
@@ -113,3 +113,33 @@ func PublicIPs() ([]string, error) {
 
 	return ips, nil
 }
+
+// PrivateIP returns a first matching IP that belong to the given network CIDR.
+func PrivateIP(networkCIDR string) (string, error) {
```

Comment:
The parameter can be called simply `cidr`.

Reply:
Renamed...